packages/atomic/src/components/search/tabs/atomic-tab-manager/atomic-tab-manager.pcss

```diff
@@ -1,11 +1,13 @@
 @import '../../../../global/global.pcss';
 
-@media (max-width: 768px) {
-  .tab-dropdown-area {
-    display: block;
-  }
+.hide-tabs {
```

Comment:
Why not `hidden`?

Reply:
Hidden would break the dynamic dropdown. We need the element to have a width so we can check if there's enough space for it and display the dropdown when not.

Reply:
I mean why not name the class "hidden"? 😁

Reply:
hidden is a tailwind class already https://tailwindcss.com/docs/display